python/paddle/fluid/framework.py

```diff
@@ -1389,6 +1389,14 @@ def clone(self, for_test=False):
         Notes: This API DOES NOT prune any operator. Use
         :code:`clone(for_test=True)` before backward and optimization please.
 
+        .. code-block:: python
+
+            // clone(for_test=True) must be used before optimization please.
+            // Otherwise, the program will remain the same as the training phase.
```

Comment:
I think the follow is better.
```
        NOTES:
            This API **DOES NOT** prune any operator. Use :code:`clone(for_test=True)`
            before backward and optimization please, e.g.

                >>> test_program = fluid.default_main_program().clone(for_test=True)
                >>> optimizer = fluid.optimizer.Momentum(learning_rate=0.01, momentum=0.9)
                >>> optimizer.minimize()
```
<img width="711" alt="clone" src="https://user-images.githubusercontent.com/30176695/41770869-076c2890-7646-11e8-8c62-6b259804f1f0.png">

Reply:
Done